mainapp/helpers.py

```diff
@@ -112,7 +113,7 @@ def genre_wise(genre, percentile=0.85):
     qualified = qualified.assign(weighted_rating=W)
     qualified.sort_values('weighted_rating', ascending=False, inplace=True)
 
-    return qualified[cols].head(min_genre_book_count).sample(n_books)
+    return qualified[cols].head(min_genre_book_count).sample(int(2*n_books))
```

Comment:
why `int`, let's pass an already converted `integer` value instead of leaving it to the function